karate-core/src/main/java/com/intuit/karate/UICookieUtils.java

```diff
@@ -0,0 +1,33 @@
+package com.intuit.karate;
+
+import com.intuit.karate.http.Cookie;
+
+import java.util.HashMap;
+import java.util.Map;
+
+/**
+ * Cookie Utility method for UI calls.
+ */
+public class UICookieUtils {
```

Comment:
I think you can add this utility method to`com.intuit.karate.http.HttpUtils`

Reply:
done!